velox/exec/tests/TableWriterTest.cpp

```diff
@@ -315,6 +315,50 @@ class UnpartitionedTableWriterTest
   }
 };
 
+class BucketedUnpartitionedTableWriterTest
+    : public TableWriterTestBase,
+      public testing::WithParamInterface<uint64_t> {
+ public:
+  BucketedUnpartitionedTableWriterTest() : TableWriterTestBase(GetParam()) {}
+
+  static std::vector<uint64_t> getTestParams() {
+    std::vector<uint64_t> testParams;
+    const std::vector<bool> multiDriverOptions = {false, true};
+    std::vector<FileFormat> fileFormats = {FileFormat::DWRF};
+    if (hasWriterFactory(FileFormat::PARQUET)) {
+      fileFormats.push_back(FileFormat::PARQUET);
+    }
+    const std::vector<TestMode> bucketModes = {TestMode::kOnlyBucketed};
+    for (bool multiDrivers : multiDriverOptions) {
+      for (FileFormat fileFormat : fileFormats) {
+        for (auto bucketMode : bucketModes) {
```

Comment:
nit: Do we need the bucketModes vector and this loop here if the vector has only one value?

Reply:
@majetideepak Good catch. I have fixed. And can you help to review again? Thanks.